src/renderer/components/message/MessageListAndComposer.js

```diff
@@ -17,22 +19,31 @@ export default function MessageListAndComposer (props) {
   const { chat } = props
   const conversationRef = useRef(null)
   const refComposer = useRef(null)
+  const { openDialog } = useContext(ScreenContext)
 
   const setComposerSize = size => setState({ composerSize: size })
 
   const onDrop = (e) => {
     const files = e.target.files || e.dataTransfer.files
     e.preventDefault()
     e.stopPropagation()
+    const tx = window.translate
+
     // TODO maybe add a clause here for windows because that uses backslash instead of slash
     const forbiddenPathRegEx = /DeltaChat\/[\d\w]*\/db\.sqlite-blobs\//gi
     for (let i = 0; i < files.length; i++) {
-      const { path } = files[i]
+      const { path, name } = files[i]
       if (!forbiddenPathRegEx.test(path.replace('\\', '/'))) {
-        callDcMethod(
-          'messageList.sendMessage',
-          [chat.id, null, path]
-        )
+        openDialog('ConfirmationDialog', {
+          message: tx('ask_send_file_desktop', [name, chat.name]),
+          cb: yes => {
+            if (!yes) { return }
```

Comment:
`{ }` are not needed here
for the sending part we should use the chat store as soon as it's merged. The call would be `chatListDispatch({type:'SEND_MESSAGE', [chat.id, null, path])`